.github/CheckingAccount.java

```diff
@@ -0,0 +1,27 @@
+public class CheckingAccount
+{
+    private int balance;
+    
+    
+    public int CheckingAccount()
+    {
+        return balance = 0;
+    }
+    
+    public int deposit(int x)
+    {
+        return balance += x;
+    }
+    
+    public int withdraw(int x)
```

Comment:
This should actually be altering the balance, not just returning what the amount would be if you subtracted that amount.